riocli/auth/staging.py

```diff
@@ -62,16 +60,13 @@ def environment(ctx: click.Context, interactive: bool, name: str):
         )
         return
 
-    # Since credentials are retained, try fetching a token with the same.
-    email = ctx.obj.data.get('email_id', None)
-    password = ctx.obj.data.get('password', None)
-    ctx.obj.data['auth_token'] = get_token(email, password)
-
-    organization = select_organization(ctx.obj)
-    select_project(ctx.obj, organization=organization)
+    ctx.obj.data['email_id'] = None
+    ctx.obj.data['auth_token'] = None
 
     ctx.obj.save()
 
+    click.secho(f'Your environment is set to {name}. Please login again using `rio auth login`', fg=Colors.GREEN)
```

Comment:
thought: This will be inconvenient for developers. We will explore adding libsecret support to read password from the keyring.